### PR TITLE
Update client-list.json (Rob / Folding @ Home)

### PR DESCRIPTION
**Pull request by RobQuistNL**
Landsat 8 is freed up for someone else - I'm taking `foldingathome-covid19` - but it exists in 4 different S3 buckets.

Since I'm tracking those independently I suggest I add it as 4 seperate datasets.

### CODE REVIEW

**Comment by dkkapur** [comment]
@RobQuistNL if we do this - we'll need to adjust your CID lists to be in 4 different .csv's that match these dataset slug names. Can you give me a 👍🏽 on this before we proceed?

**Comment by RobQuistNL** [comment]
@dkkapur done!

**Comment by dkkapur** [comment]
@orvn or @timelytree will await your 👍🏽 before proceeding as well to make sure this will flow through into the dashboard correctly

**Comment by RobQuistNL** [comment]
Already 👍'd :)

File: recovery/files/dataset-list.json [hunk 1]
```diff
@@ -8,6 +8,10 @@
   "uc-berkeley-computer-science-courses": "UC Berkeley Computer Science Courses",
   "the-librivox-free-audiobook-collection": "The LibriVox Free Audiobook Collection",
   "folding-at-home-covid": "Folding@home COVID-19 Dataset",
```
Comment (RobQuistNL) [inline recovery/files/dataset-list.json:10]: I have left this one here on purpose (backwards / db compatibility?)

**Comment by dkkapur** [comment]
@RobQuistNL - want to double check with the folks that own the backend for the app/dashboard. Hopefully will be unblocked shortly!

**Comment by RobQuistNL** [comment]
🤦 I misread your comment. I thought the "your 👍 " was directed towards me, but it was directed towards the 2 people you mentioned. Haha 🤡

File: recovery/files/dataset-list.json (hunk 1 above)
Comment (dkkapur) [reply recovery/files/dataset-list.json:10]: reasonable. lets ship this and see what happens.

**Review by dkkapur — APPROVED** [review]

**Comment by dkkapur** [comment]
Approving + merging this for now. Can revert / adjust if does not work as expected.

**Pull request merged**